rgl.gemspec

```diff
@@ -1,28 +1,28 @@
 $:.unshift File.join(File.dirname(__FILE__), 'lib')
-require 'rgl/base'
+require 'rgl/version'
 
 Gem::Specification.new do |s|
   s.name    = 'rgl'
-  s.version = RGL_VERSION
+  s.version = RGL::VERSION
   s.summary = "Ruby Graph Library"
   s.description = "RGL is a framework for graph data structures and algorithms"
-  s.licenses = ['ruby']
+  s.licenses = ['Ruby']
 
   #### Dependencies and requirements.
 
   s.add_dependency 'stream',     '~> 0.5.3'
-  s.add_dependency 'pairing_heap', '>= 0.3.0'
+  s.add_dependency 'pairing_heap', '~> 0.3'
```

Comment:
@monora This causes a downgrade from v3.0.1 to v0.3.0. Was that intentional? 🤔

Reply:
> @monora This causes a downgrade from v3.0.1 to v0.3.0. Was that intentional?
No, but it is fixed by PR #119. I will release a new version containing the fix. Thank you!